src/validation.h

```diff
@@ -831,9 +831,9 @@ class ChainstateManager
 
     //! If true, the assumed-valid chainstate has been fully validated
     //! by the background validation chainstate.
-    bool m_snapshot_validated{false};
+    bool m_snapshot_validated GUARDED_BY(::cs_main) {false};
 
-    CBlockIndex* m_best_invalid;
+    CBlockIndex* m_best_invalid GUARDED_BY(::cs_main) {nullptr};
```

Comment:
nit in the second commit:

clang format:

```diff
diff --git a/src/validation.h b/src/validation.h
index caca7f2531..adefc706fd 100644
--- a/src/validation.h
+++ b/src/validation.h
@@ -831,9 +831,9 @@ private:
 
     //! If true, the assumed-valid chainstate has been fully validated
     //! by the background validation chainstate.
-    bool m_snapshot_validated GUARDED_BY(::cs_main) {false};
+    bool m_snapshot_validated GUARDED_BY(::cs_main){false};
 
-    CBlockIndex* m_best_invalid GUARDED_BY(::cs_main) {nullptr};
+    CBlockIndex* m_best_invalid GUARDED_BY(::cs_main){nullptr};
     friend bool node::BlockManager::LoadBlockIndex(const Consensus::Params&);
 
     //! Internal helper for ActivateSnapshot().

Reply:
Done in fd6e94252e0860831527293fe9aa743044ca635f